src/sentry/integrations/opsgenie/integration.py

```diff
@@ -51,6 +60,61 @@
 )
 
 
+class InstallationForm(forms.Form):
+    base_url = forms.CharField(
+        label=_("Base URL"),
+        help_text=_("Either https://api.opsgenie.com/ or https://api.eu.opsgenie.com/"),
+        widget=forms.TextInput(attrs={"placeholder": "https://api.opsgenie.com/"}),
+    )
```

Comment:
Alternatively, since OpsGenie is a Cloud solution right now, we can make this a hardcoded list of options (e.g. ["https://api.opsgenie.com", "https://api.eu.opsgenie.com"]) and have a dropdown selector instead? What do you think?